host-guest/cb7-b2/setup-ops.py

```diff
@@ -0,0 +1,181 @@
+print storage.save(engine)
+print storage.save(engine_hot)
+
+#
+platform = engine.simulation.context.getPlatform()
+print platform.getName()
+platform = engine_hot.simulation.context.getPlatform()
+print platform.getName()
+
+# Equilibrate
+engine.current_snapshot = template
+engine.minimize()
+initial_snapshot_cool = engine.current_snapshot
+
+engine_hot.current_snapshot = template
+engine_hot.minimize()
+initial_snapshot_hot = engine_hot.current_snapshot
+
+storage.tag['cool_template'] = engine.current_snapshot
+storage.tag['hot_template'] = engine_hot.current_snapshot
+
+# Define order parameters
+# Compute the closest heavy atom distances between receptor and ligand
+cv = paths.MDTrajFunctionCV(
+    name="compute_contacts",
+    cv_scalarize_numpy_singletons=False, # needed because compute_contacts() does not return a single numpy array
+    contacts=[[0,1]],
+    f=md.compute_contacts,
+    topology=template.topology,
+    scheme='closest-heavy',
+    ignore_nonprotein=False,
+    periodic=True
+).with_diskcache()
+storage.save([cv])
+
+# Create CV states for bound and unbound
+def compute_cv(snapshot, center, compute_contacts):
+    from simtk import unit
+    distances = cv(snapshot)
+    distance = distances[0][0] * 10 # convert from nanometers to angstroms
+    #print('%8.3f' % distance)
+    return distance
+
+# State definitions
+states = [
+    'bound  ',
+    'unbound']
+
+max_bound   = 3.0 # angstroms, maximum bound state separation distance
+min_unbound = 7.0 # angstroms, minimum unbound state separation distance
+
+print('Creating interfaces...')
+ninterfaces = 30
+# note that we just changed CVRangeVolume => CVDefinedVolume, but you haven't updated that yet
+bound = paths.CVRangeVolume(cv, lambda_min=0.0, lambda_max=max_bound)
+unbound = paths.CVRangeVolume(cv, lambda_min=min_unbound, lambda_max=float("inf"))
+interfaces = paths.VolumeInterfaceSet(cv, minvals=0.0, maxvals=np.linspace(3.1, 6.9, ninterfaces))
+
+print('Creating network...')
+mistis = paths.MISTISNetwork([(bound, interfaces, unbound)])
+
+initial_trajectory_method = 'high-temperature'
+if initial_trajectory_method == 'high-temperature':
+    # generate high-temperature trajectory
+    print('Generating high-temperature trajectory...')
+    ensemble = paths.AllOutXEnsemble(bound) | paths.AllOutXEnsemble(unbound)
+    long_trajectory = engine_hot.generate(initial_snapshot_hot, [ensemble])
+    # split out the subtrajectory of interest
+    tmp_network = paths.TPSNetwork(bound, unbound)
+    short_trajectory = tmp_network.all_ensembles[0].split(long_trajectory)
+    initial_trajectories = [ short_trajectory ]
+
+elif initial_trajectory_method == 'ratchet':
+    print('Bootstrapping initial trajectory...')
+    bootstrap = paths.FullBootstrapping(
+        transition=mistis.transitions[(bound, unbound)],
+        snapshot=initial_snapshot_hot,
+        engine=engine_hot
+    )
+    bootstrap.run()
+    initial_trajectories = [ s.trajectory for s in list(bootstrap) ]
```

Comment:
```python
initial_sample_set = bootstrap.run()
initial_trajectories = [s.trajectory for s in initial_sample_set]
```

I think that will fix the question in https://github.com/choderalab/ligand-binding-pathsampling/pull/3#issuecomment-237857787